ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -29,6 +29,10 @@ if [[ "${CLUSTER_TYPE}" == "azurestack" ]]; then
   export AZURE_AUTH_LOCATION=$SHARED_DIR/osServicePrincipal.json
 fi
 
+if [[ "${CLUSTER_TYPE}" == "azuremag" ]]; then
+  export AZURE_AUTH_LOCATION=$SHARED_DIR/osServicePrincipal.json
```

Comment:
After my update for `osServicePrincipal.json` key in vault, I think we do NOT need to change `AZURE_AUTH_LOCATION` any more, directly use the file at the default location of `$CLUSTER_PROFILE_DIR/osServicePrincipal.json`.